Description: disable the test expecting a timeout 
Author: Santiago Vila <sanvila@debian.org>
Origin: vendor
Bug-Debian: https://bugs.debian.org/928505
Forwarded: no
Last-Update: 2019-05-06

Index: ruby-pygments.rb/test/test_pygments.rb
===================================================================
--- ruby-pygments.rb.orig/test/test_pygments.rb
+++ ruby-pygments.rb/test/test_pygments.rb
@@ -38,12 +38,6 @@ class PygmentsHighlightTest < Test::Unit
     assert_equal 458511, code.bytesize.to_i
   end
 
-  def test_returns_nil_on_timeout
-    large_code = REDIS_CODE * 300
-    code = P.highlight(large_code) # a 30 mb highlight request will timeout
-    assert_equal nil, code
-  end
-
   def test_highlight_works_with_null_bytes
     code = P.highlight("\0hello", :lexer => 'rb')
     assert_match "hello", code
