From: Antonio Terceiro <terceiro@debian.org>
Date: Sun, 5 Apr 2020 10:42:16 -0300
Subject: pygments/popen: fix warning on ruby2.7

---
 lib/pygments/popen.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/pygments/popen.rb b/lib/pygments/popen.rb
index 3623d6b..98d1614 100644
--- a/lib/pygments/popen.rb
+++ b/lib/pygments/popen.rb
@@ -153,7 +153,7 @@ module Pygments
         # use upstream file path during build, use shipped file once installed
         upstream_file = File.expand_path('../../../lexers', __FILE__)
         debian_file = '/usr/share/ruby-pygments.rb/lexers'
-        lexer_file = File.exists?(debian_file) ? debian_file : upstream_file
+        lexer_file = File.exist?(debian_file) ? debian_file : upstream_file
         raw = File.open(lexer_file, "rb").read
         Marshal.load(raw)
       rescue Errno::ENOENT
